### DISCUSSION TITLE
How to output TimeDerivative value instead of output residual of it?

### DISCUSSION THREAD

**Discussion by ZhigangPu** [idaholab/moose#19292] [Q&A General] [answered]
Dear,

I wonder how to output time derivative like dx/dt instead of directly using the `save_in` parameter to output residual of the time derivative kernel?

Thanks
zhigang

**Comment by GiudGiud** [comment, marked as answer]
Hello

Not sure we have that somewhere. But you could use an auxkernel to save that in an auxvariable.
Just use coupledDot routine. 

Guillaume

**Reply by ZhigangPu** [reply]
Thanks, I will try that.
